### PR TITLE
Classic Table - Add support for tooltips and rendering styled contents

### PR DESCRIPTION
**Pull request by mojith**
Fix:

Screenshot from 2022-07-27 09-23-07

## Checklist

- [x] Test Coverage is 100% for the newly added code
- [x] Storybook stories are added/updated for the changed areas
- [ ] Components standards defined [in this document](https://docs.google.com/document/d/16_Nd3VxE_lTD9pVkONQ0egn7IiwyX1pVXZjzl-V4tU8/) are followed
- [x] Code is linted properly
- [x] Developer testing is done for the affected areas
- [ ] Package version updated (not applicable to ent branch)
- [x] Changelog.md updated on possible breaking (applicable to ent branch)

### CODE REVIEW

File: components/src/core/components/Table/ClassicTable.vue [hunk 1]
```diff
@@ -49,7 +49,20 @@
...
           :style="{width: header.width}"
         >
-          {{ item[header.name] }}
+
+        <div v-if="header.isHtml" v-html="item[header.name]"></div>
+        <div v-else>
+          <div
+            v-if="item[header.name].length > header.maxCharacterLength"
+            :tooltip="item[header.name]"
+            :flow="header.flow?header.flow:'bottom'"
```
Comment (gayan-orangehrm) [inline components/src/core/components/Table/ClassicTable.vue:58]: We can also use 
`header.flow || 'bottom'`
instead the ternary to make it much simpler

File: components/src/core/components/Table/__tests__/classic-table.spec.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            {
+              col1: 'Data 3',
+              col2: '<div style="border: solid 1px #D3D3D3; border-radius: 0.5rem; padding: 5px 10px; width: fit-content; margin-left: 10px;">Action</div>',
+              col3:
+                "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+            },
+          ]
+        },
+    });
+    expect(wrapper.html()).toMatchSnapshot();
```
Comment (gayan-orangehrm) [inline components/src/core/components/Table/__tests__/classic-table.spec.ts:25]: Can we test the props and the new features added instead the snapshot rendering

**Review by gayan-orangehrm — CHANGES_REQUESTED** [review]
@mojith
PTAL

File: components/src/core/components/Table/ClassicTable.vue (hunk 1 above)
Comment (mojith) [reply components/src/core/components/Table/ClassicTable.vue]: Changed as suggested

File: components/src/core/components/Table/__tests__/classic-table.spec.ts (hunk 2 above)
Comment (mojith) [reply components/src/core/components/Table/__tests__/classic-table.spec.ts:25]: added some more asserts

**Review by rtamarasinghe — APPROVED** [review]

**Pull request merged**